src/GitVersion.Core.Tests/Configuration/IgnoreConfigurationTests.cs

```diff
@@ -20,12 +19,12 @@ public void CanDeserialize()
 ";
 
         using var reader = new StringReader(yaml);
-        var config = ConfigSerializer.Read(reader);
+        var configuration = ConfigurationSerializer.Read(reader);
 
-        config.Ignore.ShouldNotBeNull();
-        config.Ignore.ShAs.ShouldNotBeEmpty();
-        config.Ignore.ShAs.ShouldBe(new[] { "b6c0c9fda88830ebcd563e500a5a7da5a1658e98" });
-        config.Ignore.Before.ShouldBe(DateTimeOffset.Parse("2015-10-23T12:23:15"));
+        configuration.Ignore.ShouldNotBeNull();
+        configuration.Ignore.ShAs.ShouldNotBeEmpty();
```

Comment:
Huh, I've never noticed this before. Why is `ShAs` written with an uppercase `A`? 🧐

Reply:
Done